tools/circt-test/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(LLVM_LINK_COMPONENTS Support)
+get_property(dialect_libs GLOBAL PROPERTY CIRCT_DIALECT_LIBS)
+
+add_circt_tool(circt-test circt-test.cpp)
+target_link_libraries(circt-test
+  PRIVATE
+  ${dialect_libs}
```

Comment:
Since this is a specialized tool, do we really want to link against all CIRCT dialects? Will the tool consume CIRCT core IR or also incorporate the frontend language pipelines?

Reply:
Ah great point. No I guess it's going to be mostly core dialects.

Reply:
Fixed.